### PR TITLE
Adds support for ethernet networks

### PR DESCRIPTION
**Pull request by andreadean5**
Patch doesn't support bulk requests, associatedProfiles or associatedUplinkGroups. I will add support for those in another pull request.

### CODE REVIEW

File: hpOneView/resources/networking/ethernet_networks.py [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        self._connection = con
+        self._client = ResourceClient(con, self.URI)
+        self.__default_values = {
+            "vlanId": None,
+            "purpose": "General",
+            "smartLink": False,
+            "privateNetwork": False,
+            "connectionTemplateUri": None,
+            "ethernetNetworkType": "Untagged",
+            "type": "ethernet-networkV3"
```
Comment (balestrinc) [inline hpOneView/resources/networking/ethernet_networks.py]: We only add default values to the properties that have defined a default value on the RestAPI documentation. E.g.: property "type" has a default, but "vlanId" doens't
image

File: hpOneView/resources/networking/ethernet_networks.py [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+        Returns: Created resource. When blocking=False, returns the task.
+
+        """
+        data = self.__default_values.copy()
+        data.update(resource)
+        return self._client.create(data, blocking)
+
+#     # TODO: this
+#     def create_bulk(self, resource, blocking=True):
```
Comment (balestrinc) [inline hpOneView/resources/networking/ethernet_networks.py]: I suggest not commit commented code

**Comment by GustavoHennig** [comment]
Hi @andreadean5,
You did a great work, thank you!
I just suggest you to use rebase instead of merge, it will be easier.  ;)

File: hpOneView/resources/networking/ethernet_networks.py (hunk 1 above)
Comment (andreadean5) [inline hpOneView/resources/networking/ethernet_networks.py]: Thank you! That makes lots of sense

File: hpOneView/resources/networking/ethernet_networks.py (hunk 2 above)
Comment (andreadean5) [inline hpOneView/resources/networking/ethernet_networks.py]: Got it!

**Pull request merged**